proto/viam/service/slam/v1/slam.proto

```diff
@@ -66,3 +89,42 @@ message GetMapResponse {
   // Actual MIME type of response (image/jpeg or image/pcd)
   string mime_type = 3;
 }
+
+message GetPositionNewRequest {
+  // Name of slam service
+  string name = 1;
+  // Additional arguments to the method
+  google.protobuf.Struct extra = 99;
```

Comment:
Do we need `extra` for `GetPosition`? If not, we may be able to exclude it here, since we're already making a breaking change to `GetPosition`.

Reply:
In the current proto, the field extra appears in the `GetPositionRequest`, `GetPositionResponse` and `GetMapResponse`. It is currently only being used in the `GetPositionResponse` to return the quaternion (although once the spatial math lib is available in rust, if not already, we will not need it anymore.

Since this field was not discussed in the tech spec I kept them in. However, we can freely remove it from the other two proto messages if desired.

Reply:
I think we can remove `extra` from `GetPositionNewRequest` (which will eventually become `GetPositionRequest`), since we are already making a breaking change to `GetPosition`. I think we don't need to include `extra` in `GetPointCloudMapResponse`.